corehq/apps/reports/models.py

```diff
@@ -268,7 +269,11 @@ def _dispatcher(self):
             if dispatcher.prefix == self.report_type:
                 return dispatcher()
 
-        raise Exception("Unknown dispatcher: %s" % self.report_type)
+        logging.error("This saved-report (id: %s) is unknown (report_type: %s). (Might be a legacy report)" % (
```

Comment:
The exception is handled by the other callers, I just renamed it. I was going to capture it on line 514, but it no longer calls this function, but I kept the name change just to be more clear. 

Re.logging - I added it as thinking it's a good practice. In case, we capture and consolidate all error logs somewhere in future, this won't be missed. But I will remove it if that sounds wrong.